feature-detects/websockets.js

```diff
@@ -25,5 +25,9 @@
 }
 !*/
 define(['Modernizr'], function(Modernizr) {
-  Modernizr.addTest('websockets', 'WebSocket' in window && window.WebSocket.CLOSING === 2);
+  try {
+    Modernizr.addTest('websockets', 'WebSocket' in window && window.WebSocket.CLOSING === 2);
+  } catch (e) {
+    Modernizr.addTest('websockets', false);
```

Comment:
Rather than running `addTest` twice, use this pattern

```
var supports;

try {
  supports = blah in window
} catch (e) {
  supports = false
}

Modernizr.addTest('blah', supports);
```